gazebo_ros/launch/gzclient.launch.py

```diff
@@ -44,33 +45,43 @@ def generate_launch_description():
     if 'GAZEBO_RESOURCE_PATH' in environ:
         media += ':'+environ['GAZEBO_RESOURCE_PATH']
 
-    env = {'GAZEBO_MODEL_PATH': model,
-           'GAZEBO_PLUGIN_PATH': plugin,
-           'GAZEBO_RESOURCE_PATH': media}
-
-    prefix = PythonExpression(['"gdb -ex run --args" if "true" == "',
-                                     LaunchConfiguration('gdb'),
-                                     '"else "valgrind" if "true" == "',
-                                     LaunchConfiguration('valgrind'),
-                                     '"else ""']),
+    env = {
+        'GAZEBO_MODEL_PATH': model,
+        'GAZEBO_PLUGIN_PATH': plugin,
+        'GAZEBO_RESOURCE_PATH': media}
```

Comment:
```suggestion
        'GAZEBO_RESOURCE_PATH': media
    }
```

Reply:
Done in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1110/commits/c18079fd9218ad93718fe38e0b909686e5d81a4d.